src/lib/y2partitioner/devicegraph.rb

```diff
@@ -0,0 +1,58 @@
+module Y2Partitioner
+  # Singleton class that helps to work with a copy of the system
+  # {Y2Storage::Devicegraph}
+  class Devicegraph
```

Comment:
Shouldn't this be `Devicegraphs` in plural? An instance of the class does not represent a single devicegraph, but a singleton object containing several devicegraphs. I'm not a fan of class names in plural, but I like even less class names that don't match expectations.